node/src/chain_spec/neumann.rs

```diff
@@ -7,16 +7,16 @@ use sp_core::{crypto::UncheckedInto, sr25519};
 
 use super::TELEMETRY_URL;
 use crate::chain_spec::{
-	get_account_id_from_seed, get_collator_keys_from_seed, validate_allocation, Extensions,
+	get_account_id_from_seed, get_collator_keys_from_seed, validate_allocation, validate_vesting, Extensions,
 };
 use neumann_runtime::{
-	CouncilConfig, SudoConfig, ValveConfig, DOLLAR, EXISTENTIAL_DEPOSIT, TOKEN_DECIMALS,
+	CouncilConfig, SudoConfig, ValveConfig, VestingConfig, DOLLAR, EXISTENTIAL_DEPOSIT, TOKEN_DECIMALS,
 };
 use primitives::{AccountId, AuraId, Balance};
 
 static TOKEN_SYMBOL: &str = "NEU";
 const SS_58_FORMAT: u32 = 51;
-const TOTAL_TOKENS: u128 = DOLLAR * 1_000_000_000;
+const TOTAL_TOKENS: u128 = DOLLAR * 990_000_000;
```

Comment:
if we restart the neuman chain, this would be the default right?

Reply:
i'd make one fn for staging/live and one for testing specfically to test vesting

Reply:
so for staging/live, do you want any vesting in there at all? or should we just have vesting in the test vesting one?

Reply:
yup yup, i left latest (the live one) the way it is - changed staging config to have vesting, since we're not using that one